### PR TITLE
Exclude column groups from column select in advanced filtering dialog - 8.2.x

### LINKED ISSUE #5886: Advanced Filtering Dialog should not display column groups
## Description  
Advanced Filtering Dialog should not display column groups.

 * igniteui-angular version: 8.2.x
 * browser: NA

## Steps to reproduce  

1. Open https://stackblitz.com/edit/angular-hzoxyv.
2. Open the Advanced Filtering Dialog.
3. Add a group and a condition.

## Result  
The column dropdown contains the column groups.

## Expected result  
The column dropdown should not contain column groups.

### PR DESCRIPTION
**Pull request by tachojelev**
Closes #5886
### Additional information (check all that apply):
 - [x] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [x] This PR includes unit tests covering all the new code
 - [ ] This PR includes API docs for newly added methods/properties
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/grids/grid/grid-filtering-advanced.spec.ts [hunk 1]
```diff
@@ -2662,6 +2664,35 @@ describe('IgxGrid - Advanced Filtering', () => {
             }));
         });
     });
+
+    describe('', () => {
+        let fix, grid: IgxGridComponent;
+        beforeEach(fakeAsync(() => {
+            resizeObserverIgnoreError();
```
Comment (DiyanDimitrov) [inline projects/igniteui-angular/src/lib/grids/grid/grid-filtering-advanced.spec.ts]: There is no need to call `resizeObserverIgnoreError()` anymore. It is called in `configureTestSuite`.

File: projects/igniteui-angular/src/lib/grids/grid/grid-filtering-advanced.spec.ts [hunk 2]
```diff
@@ -2662,6 +2664,35 @@ describe('IgxGrid - Advanced Filtering', () => {
...
+            addAndGroupButton.click();
+            tick(100);
+            fix.detectChanges();
+
+            // Open column dropdown and verify that there are no column groups present.
+            GridFunctions.clickAdvancedFilteringColumnSelect(fix);
+            fix.detectChanges();
+            const dropdownValues = GridFunctions.getAdvancedFilteringSelectDropdownItems(fix).map((x: any) => x.innerText);
+            const expectedValues = ['ID', 'ProductName', 'Downloads', 'Released', 'ReleaseDate', 'Another Field'];
+            verifyEqualArrays(dropdownValues, expectedValues);
```
Comment (DiyanDimitrov) [inline projects/igniteui-angular/src/lib/grids/grid/grid-filtering-advanced.spec.ts]: You could directly compare arrays with `expect().toEqual()`.

File: projects/igniteui-angular/src/lib/grids/grid/grid-filtering-advanced.spec.ts (hunk 1 above)
Comment (tachojelev) [reply projects/igniteui-angular/src/lib/grids/grid/grid-filtering-advanced.spec.ts]: Done.

File: projects/igniteui-angular/src/lib/grids/grid/grid-filtering-advanced.spec.ts (hunk 2 above)
Comment (tachojelev) [reply projects/igniteui-angular/src/lib/grids/grid/grid-filtering-advanced.spec.ts]: Done.

**Pull request merged**